roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -286,7 +306,14 @@ spec:
           name: '{{ ansible_operator_meta.name }}-ee'
           imagePullPolicy: '{{ image_pull_policy }}'
           resources: {{ ee_resource_requirements }}
-          args: ['receptor', '--config', '/etc/receptor/receptor.conf']
+          args:
+            - /bin/sh
+            - -c
+            - |
+              if [ ! -f /etc/receptor/receptor.conf ]; then
+                cp /etc/receptor/receptor-default.conf /etc/receptor/receptor.conf
```

Comment:
where is this receptor-default.conf coming from? Is this something @rooftopcellist should know about for downstream?

Reply:
good point!

Reply:
@Zokormazo Thanks for thinking to ask!

I talked to @TheRealHaoLiu, he just renamed the receptor.conf that gets config-mapped.  So this file does not need to be updated in the container build repo, it's source is `roles/installer/templates/configmaps/config.yaml.j2` which will be automatically synced downstream.

Reply:
receptor-default.conf comes from configmap that's templated from config.yaml.j2 

it's the original receptor config file so it's not a new file